src/modules/vk.js

```diff
@@ -0,0 +1,81 @@
+// Vkontakte (vk.com)
+(function(hello) {
+
+	hello.init({
+
+		vk: {
+			name: 'Vk',
+
+			// See https://vk.com/dev/oauth_dialog
+			oauth: {
+				version: 2,
+				auth: 'https://oauth.vk.com/authorize',
+				grant: 'https://oauth.vk.com/access_token'
+			},
+
+			// Authorization scopes
+			scope: {
+				basic: '',
+				email: 'email',
+				offline_access: 'offline'
+			},
+
+			// Refresh the access_token
+			refresh: true,
+
+			login: function(p) {
+				p.qs.display = window.navigator &&
+					window.navigator.userAgent &&
+					/ipad|phone|phone|android/.test(window.navigator.userAgent.toLowerCase()) ? 'mobile' : 'popup';
+			},
+
+			// API Base URL
+			base: 'https://api.vk.com/method/',
+
+			// Map GET requests
+			get: {
+				me: 'users.get'
+			},
+
+			// Map POST requests
+			post: false,
+
+			wrap: {
+				me: formatUser
+			},
+
+			// No XHR
+			xhr: false,
+
+			// All requests should be JSONP as of missing CORS headers in https://api.vk.com/method/*
+			jsonp: function(p, qs) {
+
+				if (p.path === 'me') {
```

Comment:
Specific path parameters should be defined by the GET mapping. So instead of `me: 'users.get'` one would write...

```
me: function(p,cb) {
     p.query.fields = 'id,first_name,last_name,photo_max';
     cb('users.get');
}
```